src/rashdf/geom.py

```diff
@@ -488,3 +574,171 @@ def ineffective_points(self) -> GeoDataFrame:  # noqa D102
 
     def blocked_obstructions(self) -> GeoDataFrame:  # noqa D102
         raise NotImplementedError
+
+    def steady_flow_names(self) -> list:
+        """Return the profile information for each steady flow event.
+
+        Returns
+        -------
+        DataFrame
+            A Dataframe containing the profile names for each event
+        """
+        path = "Results/Steady/Output/Output Blocks/Base Output/Steady Profiles"
```

Comment:
We've been trying to stick long, oft-reused strings like this one class-level variables similar to `RasGeomHdf.GEOM_STRUCTURES_PATH`, `RasGeomHdf.FLOW_AREA_2D_PATH`, to help avoid misspelled paths, let our editors tab-complete the paths, make the things easier to modify in the future, etc.

Reply:
This was very helpful! You'll notice that I've updated the paths to follow your recommendations in the plan hdf, where the paths get long. I followed the examples you provided and the code already there.